### ISSUE TITLE
Cannot "right-click" in a measure

### ISSUE THREAD

**Issue by olin256** [Audiveris/audiveris#726]
[Problematic file](https://github.com/Audiveris/audiveris/files/14890216/ror32.zip)

Measure 5 has two main problems:

- when right-clicking to the lower staff, I'm getting an error `Error in mouseReleased java.lang.NullPointerException: Cannot invoke "org.audiveris.omr.sig.inter.StaffBarlineInter.getReferenceCenter()" because the return value of "org.audiveris.omr.sheet.PartBarline.getStaffBarline(org.audiveris.omr.sheet.Part, org.audiveris.omr.sheet.Staff)" is null`
- the notes resist assigning to any voice

Perhaps I should add that all the braces in front of the staves were inserted manually.

**Comment by hbitteur** [comment]
@olin256 
No. Audiveris is intended for printed sheet music only.
This is not the case for your example.
You'll end up manually inserting almost everything.

Just for the fun I tried it myself for a couple of measures. A bit tedious but I don't stumble on an error as you did.
Perhaps because I inserted the braces (manually of course) right after the GRID step.

Anyway, my advice is: don't waste time fighting with Audiveris on such input

**Comment by olin256** [comment]
I agree that this image is not a suitable task for Audiveris. However, if the error did not stop me, I would quite likely finish the manual work – it turns out to be actually quite convenient to put the notes "directly" on the original sheet. In the end I used MuseScore, but had to look back and forth all the time. Perhaps there's a better way to do such tasks I'm not aware of.

In any case, I encountered an error (a strange one I'd say), so I reported it. If you believe it appears only in such hopeless cases like this one, feel free to ignore it.

**Comment by hbitteur** [comment]
Of course you are free to use Audiveris as you wish. In the image at hand, at least you get the grid of staves and measures, and the original image in the background to guide you in manual insertions.

Note that the simple reloading of your .omr file led to this message:
> [ror32] Error in class org.audiveris.omr.sheet.rhythm.Measure afterReload() java.lang.IllegalStateException: PartBarline with no proper StaffBarline in {Staff id=4 left:108 right:2356}

I noticed several inters flagged as "MANUAL" resulting from the manual insertions you had made.
But, restarting the processing from scratch, I could not reproduce the error you encountered.

If you think this is worth the effort, please restart your processing from scratch with as few manual actions as possible until you encounter the stopping error. Doing so, we would have a chance to fix it.
